### PR TITLE
[Git Hub project]: add substract method

### PR DESCRIPTION
**Pull request by Sofasmile**

### CODE REVIEW

File: src/main/java/Calculator.java [hunk 1]
```diff
@@ -6,4 +6,8 @@ public int add(int a, int b) {
     public int multiply(int a, int b) {
         return a * b;
     }
+
+    public int substract(int a, int b) {
```
Comment (fedorovychh) [inline src/main/java/Calculator.java]: subtract*

Comment (DevIgork) [inline src/main/java/Calculator.java]:
```suggestion
    public int subtract(int a, int b) {
```

**Review by fedorovychh — CHANGES_REQUESTED** [review]

**Review by forzen123212 — APPROVED** [review]

**Review by DevIgork — CHANGES_REQUESTED** [review]
good job but you have small mistake

**Comment by BohdanTrue** [comment]
I think u need to use this naming `subtract`

File: src/main/java/Calculator.java (hunk 1 above)
Comment (HovorukhaBohdan) [inline src/main/java/Calculator.java]: subtract

**Review by HovorukhaBohdan — APPROVED** [review]
👍

**Pull request merged**